### PR TITLE
fix: Delete dump temporary file from service backend

### PR DESCRIPTION
**Pull request by starhcq**
Delete dump temporary file from service backend

Log: Delete dump temporary file from service backend
Bug: https://pms.uniontech.com/bug-view-241927.html

### CODE REVIEW

**Comment by feeengli** [comment]
[是否满足兼容性要求] Y
[是否满足commit提交规范] Y
[是否满足编码规范] Y
[Review结论] Pass
[Fail原因] N/A

**Comment by myk1343** [comment]
[是否满足兼容性要求] Y
[是否满足commit提交规范] Y
[是否满足编码规范] Y
[Review结论] Pass
[Fail原因] N/A

File: logViewerService/logviewerservice.cpp [hunk 1]
```diff
@@ -145,6 +145,11 @@ QString LogViewerService::readLog(const QString &filePath)
         m_process.start("/bin/bash", QStringList() << "-c" << filePath);
         m_process.waitForFinished(-1);
 
+        // 清除解压后dump临时文件
+        QString dumpFilePath = filePath;
+        dumpFilePath = dumpFilePath.replace("readelf -n ", "");
```
Comment (rb-union) [inline logViewerService/logviewerservice.cpp:150]: 建议添加文件判断和检测，仅指定临时文件可被移除。

Comment (starhcq) [reply logViewerService/logviewerservice.cpp]: 统一有clearTempFiles接口清理临时文件，不再对文件进行判断和识别

**Pull request merged**